packages/core/src/render3/component.ts

```diff
@@ -143,10 +143,10 @@ export function renderComponent<T>(
   const renderer = opts.renderer || document;
   const componentDef = componentType.ngComponentDef;
   let component: T;
-  const oldView = enterView(createViewState(-1, renderer), null);
+  const oldView = enterView(createViewState(-1, renderer, []), null);
   try {
-    elementHost(opts.host || componentDef.tag);
-    component = directiveCreate(0, componentDef.n(), componentDef);
+    elementHost(opts.host || componentDef.tag, componentDef);
+    component = directiveCreate(1, componentDef.n(), componentDef);
```

Comment:
Comment that explains why the index is 1?

(would be a ice bonus to also add a comment that says what `n` is/does)